test/lib/api.spec.jsx

```diff
@@ -128,4 +128,14 @@ describe('api spec', function() {
     expect(store.getState().theme.name).toBe('default')
     expect(store.getState().theme.overrides).toEqual({})
   })
+
+  it('should set webviewContext', function() {
+    const { setWebviewContext } = api
+    // default webviewContext is `undefined`
+    expect(store.getState().context.webviewContext).toBe(undefined)
+    setWebviewContext('foo')
+    expect(store.getState().context.webviewContext).toBe('foo')
+    setWebviewContext('bar')
+    expect(store.getState().context.webviewContext).toBe('bar')
+  })
```

Comment:
after this test, let's setWebviewContext(undefined) again

Reply:
Alright